nautobot/core/testing/views.py

```diff
@@ -1054,6 +1054,52 @@ def test_bulk_edit_form_contains_all_pks(self):
             for pk in pk_list:
                 self.assertIn(f'<input type="hidden" name="pk" value="{pk}"', response_body)
 
+        @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"])
+        def test_bulk_edit_form_contains_all_filtered(self):
+            # We are testing the intermediary step of bulk_edit with pagination applied and additional filter.
+            # i.e. "_all" passed in the form and filter using query params.
+            # Prequisity is not to have `bulk_edit_data` values in objects created by `setUpTestData`.
+            pk_list = self._get_queryset().values_list("pk", flat=True)
+            self.assertTrue(len(pk_list) > 1)
+            obj_perm = users_models.ObjectPermission(name="Test permission", actions=["change"])
+            obj_perm.save()
+            obj_perm.users.add(self.user)
+            obj_perm.object_types.add(ContentType.objects.get_for_model(self.model))
```

Comment:
Should be able to replace these lines with something like `self.add_permissions(f"{self.model._meta.app_label}.change_{self.model._meta.model_name}")`

Reply:
Thanks applied.